src/libm/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3 -DNDEBUG")
+
+set(LIBSLEEF_SRCS sleefdp.c sleefsp.c)
+
+add_library("${TARGET_LIBSLEEF}" SHARED ${LIBSLEEF_SRCS})
+
+# Always compile sleef with -ffp-contract and log at configuration time
+message(STATUS "Using option `${STRICTMATHFLAG}` to compile libsleef")
+target_compile_options(${TARGET_LIBSLEEF} PUBLIC ${STRICTMATHFLAG})
+
+# Shared library versioning
+set_target_properties(${TARGET_LIBSLEEF} PROPERTIES VERSION ${SLEEF_VERSION_MAJOR}.${SLEEF_VERSION_MINOR} SOVERSION ${SLEEF_SOVERSION})
```

Comment:
Oh - this is very good to have. An easy way to install it on a debian-based system. We should work on this on a branch based on `master`, and generate a separate pull request for it. @Naoki, do you mind creating an issue with references to the launchpad page and your requirements?